source/assets/js/modules/profile/services/profileHandler.service.js

```diff
@@ -11,7 +11,7 @@ angular.module('locApp.modules.profile.services')
 
         handler.errors = [];
 
-        var profAttributes = ["id","title","description","date","author","remark","adherence","resourceTemplates"];
+        var profAttributes = ["id","title","description","date","author","contact","remark","adherence","resourceTemplates", "source"];
```

Comment:
i think "contact" is supposed to be gone now?

Reply:
So this actually a bug fix; importing any LoC profiles into the Profile Editor will fail because the old profiles expect the contact field, adding `contact` allows the Profile to be loaded.